homeassistant/components/homeassistant/scene.py

```diff
@@ -230,7 +232,8 @@ async def create_service(call):
                     entity_id,
                 )
                 continue
-            entities[entity_id] = State(entity_id, state.state, state.attributes)
+            attributes = {**state.attributes, ATTR_SNAPSHOT_AT: dt_util.utcnow()}
+            entities[entity_id] = State(entity_id, state.state, attributes)
```

Comment:
We shouldn't add attributes to the state. We could accidentally override an existing attribute.